### ISSUE TITLE
API docs describe keepalive incorrectly

### ISSUE THREAD

**Issue by ElmerLastdrager** [eclipse/mosquitto#621]
In the [API docs](https://mosquitto.org/api/files/mosquitto-h.html#mosquitto_connect), the description of the keepalive value is wrong. It currently states:

> _keepalive_ the number of seconds after which the broker should send a PING message to the client if no other messages have been exchanged in that time.

This should read:

> _keepalive_ the number of seconds after which the **client** should send a PING message to the **broker** if no other messages have been exchanged in that time.

Relevant lines:
https://github.com/eclipse/mosquitto/blob/46630e7325a82644be1a6fef00ef30bbd52c7ef6/lib/mosquitto.h#L337
https://github.com/eclipse/mosquitto/blob/46630e7325a82644be1a6fef00ef30bbd52c7ef6/lib/mosquitto.h#L365
https://github.com/eclipse/mosquitto/blob/46630e7325a82644be1a6fef00ef30bbd52c7ef6/lib/mosquitto.h#L398
https://github.com/eclipse/mosquitto/blob/46630e7325a82644be1a6fef00ef30bbd52c7ef6/lib/mosquitto.h#L433
https://github.com/eclipse/mosquitto/blob/46630e7325a82644be1a6fef00ef30bbd52c7ef6/lib/mosquitto.h#L469

**Comment by szarta** [comment]
Agreed - keepalive is needed by client - if no messages prior to the keepalive time, ping should be sent by the client.

**Comment by PierreF** [comment]
@szarta could you please submit your fix as PR ?

**Comment by ckrey** [comment]
The API docs for `connect` have meanwhile moved to [mosquitto_connect.h](include/mosquitto/libmosquitto_connect.h) and are now correct as suggested.

This issue can be closed.
